node/src/components/chain_synchronizer/operations.rs

```diff
@@ -568,10 +671,21 @@ where
         tokio::time::sleep(ctx.config.retry_interval()).await;
     }
 
+    if peers.is_empty() {
+        warn!(
+            %height,
+            attempts_to_get_fully_connected_peers =
+                %ctx.config.max_retries_while_not_connected(),
+            "unable to fetch item as no peers available"
+        );
+    }
+
     let item = loop {
         let peer = match peers.pop() {
             Some(peer) => peer,
-            None => return Ok(None),
+            None => {
+                return Ok(None);
+            }
```

Comment:
Nit, but this change seems unnecessary. Is it clippy-related?

Reply:
No - probably I added a log statement temporarily while debugging and forgot to revert the change fully.  I'll update it.